### PR TITLE
Connected Client to Server and Database

### PR DESCRIPTION
**Pull request by MarvinHo1**
Merged Database with Client!

### CODE REVIEW

File: client/src/components/popular.jsx [hunk 1]
```diff
@@ -1,18 +1,42 @@
...
+ });
+}
+
+// (state) => {
+//     return {quantity: state.quantity + 1};
+//   });
+
+// var state = Object.assign({}, this.state);
+// this.state = data;
+// this.setState(state);
```
Comment (tinaLikesCheese) [inline client/src/components/popular.jsx]: What does the above commented out code do? Do you need it anymore? Also, don't forget to clear up console logs eventually

**Comment by tinaLikesCheese** [comment]
Looks good!

File: client/src/components/popular.jsx (hunk 1 above)
Comment (MarvinHo1) [reply client/src/components/popular.jsx]: Oh I wanted to delete that as well :).  It was one way I tried to set up the state.

Comment (MarvinHo1) [reply client/src/components/popular.jsx]: Thank You!

**Pull request merged**